src/datePicker/DatePicker.unit.tsx

```diff
@@ -64,19 +64,45 @@ describe('DatePicker', () => {
   describe('react-day-picker', () => {
     it('Should forward props to the DayPicker component', () => {
       const now = new Date()
-      const wrapper = shallow(
-        <DatePicker name="datepicker" initialDate={now} numberOfMonths={12} />,
-      )
+      const wrapper = mount(<DatePicker name="datepicker" initialDate={now} />)
       const instance = wrapper.instance()
 
-      expect(wrapper.find(DayPicker).prop('numberOfMonths')).toEqual(12)
+      expect(wrapper.find(DayPicker).prop('numberOfMonths')).toEqual(2)
       expect(wrapper.find(DayPicker).prop('selectedDays')).toEqual(now)
-      expect(wrapper.find(DayPicker).prop('initialMonth')).toEqual(now)
       expect(wrapper.find(DayPicker).prop('onDayClick')).toEqual(instance.onDayClick)
       expect(wrapper.find(DayPicker).prop('navbarElement')).toEqual(instance.renderNavbar)
       expect(wrapper.find(DayPicker).prop('captionElement')).toEqual(instance.renderCaption)
       expect(wrapper.find(DayPicker).prop('renderDay')).toEqual(instance.renderDay)
       expect(wrapper.find(DayPicker).prop('firstDayOfWeek')).toEqual(0)
     })
+
+    it('Should forward same value for selectedDays and initialMonth props to the DayPicker component', () => {
+      const initialDate = new Date()
+      initialDate.setMonth(initialDate.getMonth() + 6)
+      const wrapper = mount(
+        <DatePicker
+          name="datepicker"
+          initialDate={initialDate}
+          initialMonth={initialDate}
+          numberOfMonths={12}
+        />,
+      )
+
+      expect(wrapper.find(DayPicker).prop('numberOfMonths')).toEqual(12)
+      expect(wrapper.find(DayPicker).prop('selectedDays')).toEqual(initialDate)
+      expect(wrapper.find(DayPicker).prop('initialMonth')).toEqual(initialDate)
```

Comment:
Can you add a test on window.scrollTo. You would need to mock:
```
global.scrollTo = jest.fn()
expect(global.scrollTo).toHaveBeenCalled()
```
If you want you can mock the offsetTop so that you can even have a test like this:
```
expect(global.scrollTo).toHaveBeenCalledWith(0, topMockValue)
```